tests/pact/tc.repl

```diff
@@ -198,6 +204,14 @@
 
   (defun rtc-object-list (l:[object{schema-a}]) 1)
 
+  (defun tcmodref-spec:bool (s:string)
+    true)
+
+  (defun tc-modref:bool (m:module{tctest-iface})
+    (m::tcmodref-spec "test"))
+
+  (defun fails-tc-modref:string (m:module{tctest-iface})
+    (m::tcmodref-spec "test"))
```

Comment:
Let's add a function, say `tc-call-modref`, that calls `tc-modref` with this module as I'm worried that `toUserType'` will blow up on a non-interface modref ...

Reply:
Will do.